File: testdata/java/level-01-basic-decls/src/Main.java
public class Main {
    public static final int MAX_RETRIES = 3;

    public static boolean debug = false;

    public static String hello() {
        return "hello";
    }

    public static int add(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        System.out.println(hello());
        int result = add(1, 2);
    }
}

File: testdata/rust/level-01-basic-decls/src/main.rs
const MAX_RETRIES: u32 = 3;

static DEBUG: bool = false;

fn hello() -> &'static str {
    "hello"
}

fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn main() {
    println!("{}", hello());
    let result = add(1, 2);
}
